packages/react-server/core/util/PageUtil.js

```diff
@@ -174,6 +174,7 @@ function standardizeElements(elements) {
 		.makeArray(elements)
 		.map(e => isRootContainer(e)?flattenForRender(e):e)
 		.reduce((m, e) => m.concat(Array.isArray(e)?e:[e]), [])
+		.map(e => isTheFold(e)?markTheFold():e)
```

Comment:
If I understand correctly, this is just converting a React element to an object that looks like `{isTheFold: true}` for the purposes of switching on it during render, right?